screens/AuthScreen.tsx

```diff
@@ -1,97 +1,79 @@
-import React, { useState } from 'react';
-import { Button } from 'react-native';
+import React from 'react';
+import { TouchableWithoutFeedback, View } from 'react-native';
 
+import { FontAwesome } from '@expo/vector-icons';
 import { StackNavigationOptions } from '@react-navigation/stack';
-import { useDispatch, useSelector } from 'react-redux';
+import { Button } from 'react-native-elements';
 
 import { NavigationData } from '../navigation/AuthNavigator';
-import { login, LoginRequestStatus, logout } from '../redux/auth';
-import { RootState } from '../redux/rootReducer';
+import Colors from '../theme/Colors';
 
 import {
   StyledButtonsContainer,
-  StyledInput,
-  StyledInputContainer,
+  StyledContainer,
+  StyledLarge,
+  StyledLink,
+  StyledLoginTextContainer,
   StyledText,
-  StyledView,
+  StyledWideContainer,
 } from './AuthScreen.styled';
 
 type Props = NavigationData<'Authorization'>;
 
-const AuthScreen: React.FC<Props> = () => {
-  const [username, setUsername] = useState<string>('');
-  const [password, setPassword] = useState<string>('');
-
-  const token: string = useSelector((state: RootState) => state.auth.token);
-  const requestedUsername: string = useSelector(
-    (state: RootState) => state.auth.username
-  );
-  const requestStatus: LoginRequestStatus = useSelector(
-    (state: RootState) => state.auth.requestStatus
-  );
-
-  const isAuthenticated: boolean = !!token;
-
-  const dispatch = useDispatch();
-
+const AuthScreen: React.FC<Props> = ({ navigation }) => {
```

Comment:
I see one lacking thing in this screen. What about error message in case of unsuccessful login?

Reply:
Right now after login failed app redirect to AuthScreen. I can add some alert with basic message for now, but I have planned to do it during form validation with Formik

Reply:
It's okay, we can leave it like this.